datacats/cli/create.py

```diff
@@ -28,6 +28,7 @@ def create(opts):
   -b --bare               Bare CKAN site with no example extension
   -i --image-only         Create the environment but don't start containers
   -n --no-sysadmin        Don't prompt for an initial sysadmin user account
+  --address=IP            Address to listen on (Linux-only) [default: 127.0.0.1]
```

Comment:
I've tried to sort the options alphabetically, just 'cause.

Please merge after those two changes.